docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md

```diff
@@ -0,0 +1,27 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
```

Comment:
Please, add a hash commit which we know the model works for sure.

Reply:
Done.